src/main.ts

```diff
@@ -10,13 +10,30 @@ async function run(): Promise<void> {
     const {deployPath, version, branch} = getDeployProps(context.ref);
     core.info(`deployPath: ${deployPath}`);
 
+    const workingDirectory = core.getInput("workingDirectory");
+    const build = core.getInput("build") || "npm run build";
+    const execOptions: ExecOptions = {};
+    if (workingDirectory) {
+      execOptions.cwd = workingDirectory;
```

Comment:
I can't see `execOptions` being used here, just configured. 

I would have expected the options to be the second argument to the exec call on 24 below.

Reply:
Good catch.